test/zookeeper/zk_dn_client_test.cc

```diff
@@ -34,6 +34,22 @@ class ZKDNClientTest : public ::testing::Test {
 	ZkClientDn *client;
 };
 
+TEST_F(ZKDNClientTest, RegisterMakesWorkQueues){
+        bool exists;
+        int error_code;
+        uint64_t size;
+
+        std::vector<std::uint8_t> data(sizeof(BlockZNode));
+        std::string path = "/work_queues/replicate/" + dn_id;
+
+	system("sudo ~/zookeeper/bin/zkCli.sh rmr /work_queues");
```

Comment:
I decided removing these nodes was no longer desired, since we moved the creation of /testing/work_queues/replicate outside of the function being tested.  So fixed I guess